common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -38,8 +38,8 @@
 import tachyon.Constants;
 import tachyon.TachyonURI;
 import tachyon.conf.TachyonConf;
-import tachyon.thrift.NetAddress;
 import tachyon.util.OSUtils;
+import tachyon.worker.WorkerNetAddress;
```

Comment:
isn't this introducing a dependency from the common modules on the server modules?